tracer/test/snapshots/QuartzTestsV4.verified.txt

```diff
@@ -0,0 +1,110 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: internal,
```

Comment:
In v3 the Name was `quartz.job.execute` but now in v4 it's `internal`. Was that intentional?